packages/pyproj/meta.yaml

```diff
@@ -19,7 +19,9 @@ build:
     export PROJ_DIR=${WASM_LIBRARY_DIR}
     export PROJ_INCDIR=${WASM_LIBRARY_DIR}/include
     export PROJ_LIBDIR=${WASM_LIBRARY_DIR}/lib
-
+    export PROJ_WHEEL=1
+    mkdir -p pyproj/proj_dir
+    cp -r ${WASM_LIBRARY_DIR}/share pyproj/proj_dir
```

Comment:
I'm guessing this is the best option for now but that would duplicate all these files in projects that depend both on proj and pyproj, right?

Reply:
Probably, yes. I don't know much about libproj, but it seems like packages that use libproj all embeds `pyproj.db` inside its wheel. For example, inside [fiona](https://pypi.org/project/Fiona/), I can see `pyproj.db`.

```
$ unzip -l Fiona-1.8.21-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl  | grep proj.db
  7921664  2022-02-07 15:46   fiona/proj_data/proj.db
```

Since `proj.db` is a big file in terms of the browser (~7.7MB), it would be great if we can share it between multiple packages. Maybe someone can ask package maintainers to do such optimization.

Reply:
It is a design problem with the wheel. They a need to bundle proj and its data and we ended up with multiple copies of it. Package manager that have a more "ecosystem" logic can share data and c-lib dependencies in a more efficient way.

I'm not a pyodide expert but maybe we can use the same approach here as we do with conda?

Reply:
I think we will keep using wheels for compatibility, but that sound very interesting, thanks.